tests/test_inline.py

```diff
@@ -0,0 +1,144 @@
+import traceback
+from pybricksdev import inline
+
+
+def test_1_simple_import_from_sibling():
+    # given
+    script_path = "inline_test_resources/1/script.py"
```

Comment:
When I run `pytest` locally on Linux, I'm getting `FileNotFoundError` on all of the tests.

We should be able to use `__file__` to get the path of the current file and then use that to find other files relative to it. That way it won't matter which directory you run `pytest` in.